app/grandchallenge/evaluation/models.py

```diff
@@ -1850,6 +1860,52 @@ def inputs_complete(self):
         else:
             return False
 
+    @cached_property
+    def additional_inputs_complete(self):
+        phase_input_slugs = self.submission.phase.inputs.values_list(
+            "slug", flat=True
+        )
+        return self.inputs.filter(
+            interface__slug__in=phase_input_slugs
+        ).count() == len(phase_input_slugs)
```

Comment:
Does this work if the evaluation is rescheduled for algorithm submission evaluations where there are many inputs of the same type?